### PR TITLE
Fixed Branch Sync

### PR DESCRIPTION
**Pull request by VeXHarbinger**
Fixed my Git Config and it looks like the changes are now in sync on this branch.  This set of changes consisted of removing the depreciated Twitter Card Types (Most changes only occurred in the Twitter/Cards folder) and enhancing the comments on the Twitter Cards for the IntelliSense.  I will review the comment changes between this branch and the Extending branch to make sure the comments remain the same between the two so that the compare wont be extremely verbose when you are trying to review the code.

### CODE REVIEW

File: CodeMaid.config [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (RehanSaeed) [inline CodeMaid.config]: I guess you are using CodeMaid. This file probably doesn't need to be here. If it keeps getting added, feel free to add it to the .gitignores.

**Comment by VeXHarbinger** [comment]
Also, sorry about the excessive check-ins.  I'm sure that it's a bit cumbersome and I'll cut it out.

**Comment by RehanSaeed** [comment]
I'm useless at git branching and merging. I totally understand! Please do close any PR's that get messed up. Raising one that removes the deprecated cards is probably a good first PR.

Did you read any explanation for how and why Twitter deprecated some of their cards?

File: Source/MVC5/Boilerplate.Web.Mvc5/Twitter/Cards/PlayerTwitterCard.cs [hunk 2]
```diff
@@ -4,21 +4,25 @@
     using System.Text;
 
     /// <summary>
-    /// Video clips and audio streams have a special place on the Twitter platform thanks to the Player Card. By implementing a few HTML meta tags to
-    /// your website and following the Twitter Rules of the Road, you can deliver your rich media to users across the globe.
-    /// Twitter must approve the use of the player card, find out more below.
+    /// Video clips and audio streams have a special place on the Twitter platform thanks to the Player Card. By
+    /// implementing a few HTML meta tags to your website and following the Twitter Rules of the Road, you can deliver
+    /// your rich media to users across the globe. Twitter must approve the use of the player card, find out more below.
```
Comment (VeXHarbinger) [inline Source/MVC5/Boilerplate.Web.Mvc5/Twitter/Cards/PlayerTwitterCard.cs]: widescreen is offset by multiple parallel windows. I'll look at turning off the wrap entirely.  Personally. I prefer it not wrap.

**Comment by VeXHarbinger** [comment]
I didn't read up on the causality, just saw that it had been done.  I wanted to stay on task with making sure I caught what was changed and required fields.   I'll see about closing the PRs

**Comment by VeXHarbinger** [comment]
Ok, so do I need to issue a PR at all, or we will do a PR after the review?

**Comment by RehanSaeed** [comment]
Steps are as follows:

1. Fork this repository.
2. Create a branch in your forked repository.
3. Make changes on the branch.
4. Submit a PR based on the branch.
5. I can then take a look and make comments. If we decide to change something, you make the change on your branch and the PR gets updated auto-magically.
5. Optional - If you have several commits, you can use `git cherrypick` to squash them into a single commit.
7. I hit the merge button in GitHub and all your commits merged into the master branch of this repository. I can also hit the merge and squash button in GitHub which squashes your commits but I think it uses the latest commit message (not sure) so you lose control over that.

Hope that helps. I routinely screw the above up and only get it right through trial and error. Appreciate the contribution!

**Comment by VeXHarbinger** [comment]
Oh I'm just happy to help, both projects have been a great tool for both development and learning.
OK, So starting with the first branch, I issued the PR for the Depreciated card types.  I'll await the process to finish, then do a re-pull, sync with my extending users branch and make sure all is working as expected, then issue a PR on that branch.

**Comment by VeXHarbinger** [comment]
Howdy Muhammad,

I hope all is well on your side of the screen. I'm sure you've been busy
over the past week, I was just wondering if you are still awaiting me to
address any changes to the latest pull request?
https://github.com/ASP-NET-MVC-Boilerplate/Framework/pull/6

I believe I addressed all the items you indicated and it appears as if the
pr contains all the proper files.
Like I said I'm sure you are just busy, I'm just excited to get these two
branches merged in so that the depreciated cards are gone and the test kits
are in place so that I can address the extended image object and another
item or two.  It will just be easier after the merge so that I won't have
to address the extended image obj in the removed card types and can add the
testing of the object to the test solution.


Thanks for your time & energy,
   Alex Pearlstein

  "For we are the music makers,
       and we are the dreamers of dreams" - Willy Wonka

**Pull request closed without merging**